requirements.txt

```diff
@@ -23,7 +23,7 @@ mindsdb-sql >= 0.6.1, < 0.7.0
 mindsdb-evaluator >= 0.0.7, < 0.1.0
 checksumdir >= 1.2.0
 mindsdb-streams == 0.1.1
```

Comment:
@paxcema     

mindsdb-streams 0.1.1 depends on requests==2.26.0
chromadb 0.3.25 depends on requests>=2.28

any way to release a version of mindsdb-streams that has requests==2.28 ?

Reply:
@paxcema sqlalchemy>1.3.21,<1.4 is also causing dep issues, does it absolutely need to be pinned in that range?

Reply:
@tomhuds fyi